tools/proofers/character_selector.js

```diff
@@ -69,9 +69,14 @@ $(function () {
     }
 
     function addMru(char, titletext) {
-        var index = mru.findIndex(function (element) {
```

Comment:
Let's add this block back in, but commented out, and a comment line above it:
```
// IE HACK - IE11 doesn't support .findIndex()
```

Reply:
Done this